champ/lib/champ-wasm/lib.rs

```diff
@@ -7,10 +7,11 @@ pub struct Wallet {}
 #[allow(clippy::unused_unit)]
 #[wasm_bindgen]
 impl Wallet {
-    #[wasm_bindgen(catch, js_name = "generate")]
-    pub fn generate_wallet(password: String) -> Result<String, JsValue> {
-        lulw::generate_wallet(password).map_err(|e| JsValue::from_str(&e.to_string()))
-    }
+    //#[wasm_bindgen(catch, js_name = "generate")]
+    //pub fn generate_wallet(password: String) -> Result<(String, String), JsValue> {
+    //    lulw::generate_wallet(password).map_err(|e| JsValue::from_str(&e.to_string()))
+    //}
+    //TODO fix this
```

Comment:
I think this would be best with a rust struct which would then automatically be converted to a js object so we can 
 easily do something like `const {data, address} = generateWallet("");` on the js site https://rustwasm.github.io/wasm-bindgen/reference/types/exported-rust-types.html#example-rust-usage